src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -83,11 +83,11 @@ function initial_condition_convergence_test(x, t, equations::IdealGlmMhdEquation
     RealT = eltype(x)
     alpha = 0.25f0 * convert(RealT, pi)
     x_perp = x[1] * cos(alpha) + x[2] * sin(alpha)
-    B_perp = convert(RealT, 0.1) * sinpi(2 * x_perp)
+    B_perp = convert(RealT, 0.1) * sinpi(2 * (x_perp + t))
```

Comment:
```suggestion
    B_perp = convert(RealT, 0.1) * sinpi(2 * (x_perp - t))
```
Shouldn't this have a negative sign on the time variable? Similar to how we coded this in [FLUXO](https://github.com/project-fluxo/fluxo/blob/master/src/equation/mhd/equation.f90#L650).

The same goes for line 90 below (github won't let me leave a suggestion there for some reason).

Reply:
Thanks for checking this. I tried both variants, and `+ t` is the only one that gave the right EOC for Dirichlet BCs. I checked the FLUXO initial condition, and it seems to be a different Alfvén wave setup.